Sources/VaporToolbox/Clean.swift

```diff
@@ -76,15 +73,15 @@ class Cleaner {
         guard files.contains(".swiftpm") else {
             return .notNecessary
         }
-        try Process.shell.delete(".swiftpm")
+        try FileManager.default.removeItem(atPath: self.cwd.appendingPathComponents(".swiftpm"))
```

Comment:
I feel like all these delete calls could be refactored out into a function so the call site looks something like:

```swift
try delete(".swiftpm")
```

Reply:
At that point I might as well just put back all the `Process.shell` methods and replace their implementations 🙂 I personally don't think `FileManager` is sufficiently over-verbose to be worth it, but I'm willing to bother if you think it's significant.